client/src/components/common/footer/style.css

```diff
@@ -0,0 +1,63 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway&display=swap");
+body {
+  font-family: "Raleway", sans-serif;
+}
+.footer {
+  background-color: #eff6f5;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.container {
+  display: flex;
+  font-size: 16px;
+  text-align: left;
+  padding: 4em;
+}
+
+.footer__p1 {
+  font-size: 1em;
+  color: #000000;
+}
+
+.footer__p2 {
+  width: min(37em, calc(100% - 2vw));
+  font-size: 1em;
+  color: #000000;
+}
+
+.footer__lists {
+  display: flex;
+  text-align: left;
+  font-size: 1em;
+  line-height: 28px;
+  padding: 1em;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.footer__button {
+  background: none;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  color: #1890ff;
+  padding: 0;
+}
+
+@media screen and (max-width: 600px) {
+  .container {
```

Comment:
If we have multiple `.container` classes with different styles, they will clash in the bundle and styling bugs will occur. `.footer__something-container` would be safer unless you know `.container` will have the exact same style across the app.